pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.nuxeo</groupId>
     <artifactId>nuxeo-parent</artifactId>
-    <version>2023.4.13</version>
+    <version>2023.14</version>
```

Comment:
I could have put this diff in a separate commit referencing [NXP-32743](https://jira.nuxeo.com/browse/NXP-32743)

Reply:
I think you can add it to your Align on 2023.14 commit if you want.